src/components/ContactsList.js

```diff
@@ -41,7 +41,7 @@ class ContactsList extends React.Component<Props> {
   render() {
     return (
       <List
-        data={ContactsStore.contacts}
+        data={ContactsStore.contacts.slice()}
```

Comment:
Hmm.. I think that you should probably just duplicate the data in the `ContactsStore` so it's encapsulated.